src/Components/Post/Post-Gallery.js

```diff
@@ -4,99 +4,131 @@ import { connect } from 'react-redux';
+            || (this.props.images.length <= 1)) {
+            return null
         }
-        if (this.props.images.length > Constants.NONE_SCROLLABLE_THUMBS) {
-            return (
-                <div className="post_details postImages">
-                    <FontAwesomeIcon className="icon_angleLeft" icon="angle-left" />
-                    <div className="imageWrapper">
-                        <div className="images">
-                            {this.handleImages()}
-                        </div>
-                    </div>
-                    <FontAwesomeIcon className="icon_angleRight" icon="angle-right" /> 
+        return(
+            <div className="thumbnailNagivationButtons" 
+                onMouseOver={()=> this.toggleImageNavigationButtons(true)}
+                onMouseLeave={()=> this.toggleImageNavigationButtons(true)}>
+                <div className="thumbnailNavigationButton thumbnailNavigationButton_left"
+                    onClick={()=> this.onThumbnailButtonsClicked(true)}>
+                    <FontAwesomeIcon icon="angle-left" />
                 </div>
-            )
-        } else {
-            return (
-                <div className="post_details postImages">
-                    <div className="imageWrapper">
-                        <div className="images">
-                            {this.handleImages()}
-                        </div>
-                    </div>
+                <div className="thumbnailNavigationButton thumbnailNavigationButton_right" 
+                    onClick={()=> this.onThumbnailButtonsClicked(false)}>
+                    <FontAwesomeIcon icon="angle-right"/>   
                 </div>
-            )
-        }
+            </div>
+        )
     }
 
-    renderNote() {
-        if (!this.props.description) {
-            return null
-        }
+    renderFirstThumbnail() {
+        const imagePath = `${Constants.UPLOADS_HOSTNAME}${this.state.currentDisplayImageSrc}`;
+        return (
+            <img className="image" 
+                src={imagePath}
+                height="170" width="203"
+                alt=" "
+                onClick={(e)=> this.handleClickedImage(e.target.src, this.props.images)}>
+            </img>
+        )
+    }
+
+    toggleImageNavigationButtons(toggle) {
+        this.setState({
+            showImageNavigationButtons: toggle,
+        })
+    }
+
+    renderImages() {
         return (
-            <div className="post_details post_note">
-                {this.props.description}
+            <div className="thumbnailContainer"
+                onMouseOver={()=> this.toggleImageNavigationButtons(true)}
+                onMouseLeave={()=> this.toggleImageNavigationButtons(false)}>
+                {this.renderFirstThumbnail()}
+                {this.renderThumbnailNagivationButtons()}
+            </div>
+        )
+    }
+
+    getCity() {
+        return this.props.address.split(',')[1];
```

Comment:
Maybe `this.props.address` should be passed in as an object? like,
```
{
  city: 'asdf',
  state: 'qwer',
}
```
instead of parsing it each time